package.json

```diff
@@ -228,11 +229,11 @@
     "build": "npm run build-css && npm run build-packages && run-p -s build-devdocs:* build-server build-client-if-prod build-client-if-desktop",
     "prebuild-css": "node bin/build-custom-properties-css.js",
     "build-css": "run-p -s build-css:*",
-    "build-css:debug": "node-sass --include-path client --source-map \"public/style-debug.css.map\" assets/stylesheets/style.scss public/style-debug.css && npm run -s postcss -- public/style-debug.css && rtlcss public/style-debug.css public/style-debug-rtl.css",
-    "build-css:directly": "node-sass --include-path client assets/stylesheets/directly.scss public/directly.css --output-style compressed && npm run -s postcss -- public/directly.css",
-    "build-css:emergent-paywall": "node-sass --include-path client assets/stylesheets/emergent-paywall.scss public/emergent-paywall.css --output-style compressed && npm run -s postcss -- public/emergent-paywall.css",
-    "build-css:editor": "node-sass --include-path client assets/stylesheets/editor.scss public/editor.css --output-style compressed && npm run -s postcss -- public/editor.css",
-    "build-css:style": "node-sass --include-path client assets/stylesheets/style.scss public/style.css --output-style compressed && npm run -s postcss -- public/style.css && rtlcss public/style.css public/style-rtl.css",
+    "build-css:debug": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --include-path client --source-map \"public/style-debug.css.map\" assets/stylesheets/style.scss public/style-debug.css && npm run -s postcss -- public/style-debug.css && rtlcss public/style-debug.css public/style-debug-rtl.css",
+    "build-css:directly": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --include-path client assets/stylesheets/directly.scss public/directly.css --output-style compressed && npm run -s postcss -- public/directly.css",
+    "build-css:emergent-paywall": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --include-path client assets/stylesheets/emergent-paywall.scss public/emergent-paywall.css --output-style compressed && npm run -s postcss -- public/emergent-paywall.css",
+    "build-css:editor": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --include-path client assets/stylesheets/editor.scss public/editor.css --output-style compressed && npm run -s postcss -- public/editor.css",
+    "build-css:style": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --include-path client assets/stylesheets/style.scss public/style.css --output-style compressed && npm run -s postcss -- public/style.css && rtlcss public/style.css public/style-rtl.css",
```

Comment:
How about defining

```
"node-sass": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --include-path client",
```

so we can use it to define

```
 "build-css:directly": "npm run node-sass -- assets/stylesheets/directly.scss public/directly.css --output-style compressed && npm run -s postcss -- public/directly.css",
```

and friends?

Reply:
Or, alternatively, create a `bin/build-with-sass` script that can be called like:
```
./bin/build-with-sass --mode=production --rtl=true assets/stylesheets/directly.scss
```
That would remove a lot of verbosity and repetition from `package.json`.

The `directly`, `emergent-paywall` et all stylesheets probably won't be ever migrated to webpack. They are exposed on an agreed-upon URL for 3rd party apps rather than used by Calypso itself.

Reply:
@ockham I like the idea of removing some duplication, but I don't love overwriting the common usage of node-sass with a local node-sass npm script. It just makes it harder to copy / paste the command to the command line when debugging it or playing around with alterations. 

@jsnajdr sounds like a good plan. Want to take a shot at that in a follow up PR? I'd like to keep this one focused and clean up the import sites.

Reply:
I'm fine with that. Just sayin' that we're doing the alias `script` thing with `postcss` :grimacing: :

https://github.com/Automattic/wp-calypso/blob/d638c52938402176c0c2f47f1e61f3e74c25e082/package.json#L227